.aspect/workflows/config.yaml

```diff
@@ -1,11 +1,13 @@
 # See https://docs.aspect.build/v/workflows/config
 ---
-queue: aspect-small-arm64
+queue: aspect-nano
```

Comment:
This is set to nano as it controls the finalization step queue. In the future we should make it possible to set the finalization step queue explicitly.

Reply:
cc @mattem